### PR TITLE
Prod will now log to papertrail

### PR DESCRIPTION
**Pull request by oudeismetis**
Connects to #71

### CODE REVIEW

File: franklin/config/settings/base.py [hunk 1]
```diff
@@ -200,19 +200,26 @@
             'backupCount': LOGFILE_COUNT,
             'formatter': 'simple'
         },
+        'paper_trail': {
+            'level': 'DEBUG',
+            'filters': ['require_debug_false'],
+            'class': 'logging.handlers.SysLogHandler',
+            'formatter': 'verbose',
+            'address': ('logs.papertrailapp.com', 63032)
+        }
```
Comment (oudeismetis) [inline franklin/config/settings/base.py]: @JulianGindi - Papertrail doesn't have any security on sending logs to them. Anyone who has the above address will be able to log to us. Should this be an ENV var?

Comment (JulianGindi) [inline franklin/config/settings/base.py]: I don't think we need to worry about that for now. There are some ways we can lock it down, but I'll hop on that next sprint.

**Comment by CorneliusIV** [comment]
:+1:

**Comment by JulianGindi** [comment]
:+1:

**Pull request closed without merging**